src/Command/Issue/IssueCopyCommand.php

```diff
@@ -67,19 +67,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $adapter->setUsername($targetUsername);
         $adapter->setRepository($targetRepository);
 
-        $this->getHelper('gush_style')->success(
-            sprintf(
-                'Opened issue: %s',
-                $adapter->getIssueUrl(
-                    $adapter->openIssue(
-                        $srcTitle,
-                        $srcIssue['body'],
-                        $srcIssue
-                    )
-                )
+        $issueUrl = $adapter->getIssueUrl(
+            $adapter->openIssue(
+                $srcTitle,
+                $srcIssue['body'],
+                $srcIssue
             )
         );
 
+        $this->getHelper('gush_style')->success(
```

Comment:
Even better ;) in Gush 2.0 when using the webmozart/console we can use (and typehint) our own IO class with these methods, this is also what Composer does with its own IO class.